### PR TITLE
feat: update url to track.vote

### PR DESCRIPTION
**Pull request by beausmith**

### CODE REVIEW

File: client/src/components/HandMarkedPaperBallot.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ const ballotMetadata = ({
     ['bs', ballotStyleId],
     ['p', `${pageNumber}-${pageCount}`],
   ])
-  return new URL(`https://vx.vote/?${params}`).toString()
+  return new URL(`https://track.vote/?${params}`).toString()
```
Comment (benadida) [inline client/src/components/HandMarkedPaperBallot.tsx:162]: I don't think we want something in the *ballot* that implies tracking. I know we wouldn't be, but someone reading that QR code could get really scared. We should think about what this URL should be, but probably not `track.vote`.

Comment (beausmith) [reply client/src/components/HandMarkedPaperBallot.tsx:162]: Ok, what should url be? We don't (yet!) own `vx.vote`.

Comment (eventualbuddha) [reply client/src/components/HandMarkedPaperBallot.tsx:162]: `ballot.page`

**Comment by beausmith** [comment]
This is covered in: https://github.com/votingworks/election-manager/pull/36

**Pull request closed without merging**